### PR TITLE
Create AIPackage.md

### PR DESCRIPTION
**Pull request by rgopikrishnan91**
Created the first version of AIPackage class file. Please review this pull request 
@kestewart @zvr @lumjjb

### CODE REVIEW

File: model/AI/Classes/AIPackage.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+## Metadata
+
+- SubclassOf: Software/Package
+						Defaults are: 0..*
+## Properties
+-energyConsumption
+  - type: xsd:string
+  - minCount: 0
+  - maxCount: 1
+
```
Comment (maxhbr) [inline model/AI/Classes/AIPackage.md:19]: this and the following newlines should be dropped. The parsing is rather fragile and expects very specific structure

File: model/AI/Classes/AIPackage.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+# AIPackage
+
+## Description
+
+Metadata information that can be added to a package to describe an AI application or trained AI model. 
+
+## Metadata
+
+- SubclassOf: Software/Package
+						Defaults are: 0..*
```
Comment (maxhbr) [inline model/AI/Classes/AIPackage.md:13]: What is that? this will break parsing.

File: model/AI/Classes/AIPackage.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
+-------
```
Comment (maxhbr) [inline model/AI/Classes/AIPackage.md:1]: why is this additional line here? this might break parsing

File: model/AI/Classes/AIPackage.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+SPDX-License-Identifier: Community-Spec-1.0
+
+# AIPackage
```
Comment (maxhbr) [inline model/AI/Classes/AIPackage.md:4]: this file is missing the summary section. this might break parsing

**Review by maxhbr — CHANGES_REQUESTED** [review]
This file might not be parseable

**Comment by maxhbr** [comment]
I think that it would be better to have one PR that adds a consistent set of files, instead of 20. Otherwise, the same comments need to be added to all of them and without seeing the context and the mentioned other files it is hard to review them.

Can you please merge all PRs into one?

File: model/AI/Classes/AIPackage.md
Comment (maxhbr) [inline model/AI/Classes/AIPackage.md:1]: To make this file valid, you need to do the following changes:

```diff
diff --git a/model/AI/Classes/AIPackage.md b/model/AI/Classes/AIPackage.md
index 715697e..84e699b 100644
--- a/model/AI/Classes/AIPackage.md
+++ b/model/AI/Classes/AIPackage.md
@@ -1,4 +1,3 @@
--------
 SPDX-License-Identifier: Community-Spec-1.0
 
 # AIPackage
@@ -9,86 +8,69 @@ Metadata information that can be added to a package to describe an AI applicatio
 
 ## Metadata
 
-- SubclassOf: Software/Package
-						Defaults are: 0..*
+- name: AIPackage
+- SubclassOf: Software:Package
+- Instantiability: Concrete
+  
 ## Properties
--energyConsumption
+- energyConsumption
   - type: xsd:string
   - minCount: 0
   - maxCount: 1
-  
--standardsCompliance
+- standardsCompliance
   - type: xsd:string
   - minCount: 0
-  
--limitations
+- limitations
   - type: xsd:string
   - minCount: 0
   - maxCount:1
-
--typeOfModel
+- typeOfModel
   - type: xsd:string
   - minCount: 0
-
--informationAboutTraining
+- informationAboutTraining
   - type: xsd:string
   - minCount: 0
   - maxCount: 1
-
--informationAboutApplication
+- informationAboutApplication
   - type: xsd:string
   - minCount: 0
   - maxCount: 1
-
--hyperparameters
+- hyperparameters
   - type: xsd:string
   - minCount: 0
-
-
--dataPreprocessingSteps
+- dataPreprocessingSteps
   - type: xsd:string
   - minCount: 0
   - maxCount: 1
-
--modelExplainabilityMechanisms
+- modelExplainabilityMechanisms
   - type: xsd:string
   - minCount: 0
   - maxCount: 1
-
--sensitivePersonalInformation
+- sensitivePersonalInformation
   - type: PresenceType
   - minCount: 0
   - maxCount: 1
-
--metricsDecisionThresholds
+- metricsDecisionThresholds
   - type: xsd:string
   - minCount: 0
-
--metrics
+- metrics
   - type: xsd:string
   - minCount: 0
-
--domain
+- domain
   - type: xsd:string
   - minCount: 0
-
-
--autonomyType
+- autonomyType
   - type: PresenceType
   - minCount: 0
   - maxCount: 1
-
-
--safetyRiskAssessment
+- safetyRiskAssessment
   - type: SafetyRiskAssessmentType
   - minCount: 0
   - maxCount: 1
-
 - contentIdentifier
   - type: anyURI
   - minCount: 0
   - maxCount: 1
-
 - packagePurpose
   - type: SoftwarePurpose
   - minCount: 0
@@ -104,4 +86,3 @@ Metadata information that can be added to a package to describe an AI applicatio
   - type: anyURI
   - minCount: 0
   - maxCount: 1
-
```

**Review by maxhbr — CHANGES_REQUESTED** [review]
please make the file valid

**Comment by maxhbr** [comment]
This should not list the fields already present in `Package`

File: model/AI/Classes/AIPackage.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+  - type: xsd:string
+  - minCount: 0
+  - maxCount: 1
+
+-modelExplainabilityMechanisms
+  - type: xsd:string
+  - minCount: 0
+  - maxCount: 1
+
+-sensitivePersonalInformation
```
Comment (maxhbr) [inline model/AI/Classes/AIPackage.md:58]: the type is wrong here:

```suggestion
- sensitivePersonalInformation
  - type: SensitivePersonalInformationType
  - minCount: 0
  - maxCount: 1
```

**Review by maxhbr — CHANGES_REQUESTED** [review]

File: model/AI/Classes/AIPackage.md
Comment (rgopikrishnan91) [reply model/AI/Classes/AIPackage.md:1]: Thanks for the suggested changes. Adding them to the new version. Will also merge all the files to be added into one PR

File: model/AI/Classes/AIPackage.md (hunk 4 above)
Comment (rgopikrishnan91) [reply model/AI/Classes/AIPackage.md:4]: Thanks. Added summary to the new version

File: model/AI/Classes/AIPackage.md (hunk 3 above)
Comment (rgopikrishnan91) [reply model/AI/Classes/AIPackage.md:1]: Thanks. Fixed.

File: model/AI/Classes/AIPackage.md (hunk 2 above)
Comment (rgopikrishnan91) [reply model/AI/Classes/AIPackage.md:13]: Thanks, added metadata per your suggestion

File: model/AI/Classes/AIPackage.md (hunk 1 above)
Comment (rgopikrishnan91) [reply model/AI/Classes/AIPackage.md:19]: Thanks. Fixed per your suggestion

File: model/AI/Classes/AIPackage.md
Comment (kestewart) [inline model/AI/Classes/AIPackage.md:1]: Looks like some of the indentation/spacing needs to be fixed.

**Review by kestewart — APPROVED** [review]
Starting point for Technical Team review for merging in branch.

**Pull request merged**